common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -406,10 +406,13 @@ class @Problem
   reset_internal: =>
     Logger.log 'problem_reset', @answers
     $.postWithPrefix "#{@url}/problem_reset", id: @id, (response) =>
-      @el.trigger('contentChanged', [@id, response.html])
-      @render(response.html)
-      @updateProgress response
-      @scroll_to_problem_meta()
+      if response.success
+        @el.trigger('contentChanged', [@id, response.html])
+        @render(response.html, @scroll_to_problem_meta)
+        @updateProgress response
+        window.SR.readText(gettext('Problem state has been reset.'))
```

Comment:
Is this SR text for learners who have just clicked Reset? 
How about: "This problem has been reset."